src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -172,7 +173,7 @@
     }
 
     @ApiOperation(
-            value = "Returns the latest recall and release details for an offender",
+            value = "WARNING: This is a work in progress!  Returns the latest recall and release details for an offender", // TODO DT-337 remove warning when finished
```

Comment:
Could maybe use @ApiIgnore("WIP") so that it doesn't show in swagger documentation until it's ready?

Reply:
Good point, but I think we're intending to add the API to the swagger doc as a heads up to other teams.